itlwm.xcodeproj/project.pbxproj

```diff
@@ -886,10 +912,34 @@
 				F8A68DDB2482242E007EB376 /* iwlwifi-cc-a0-48.ucode */,
 				F8A68DD72482224F007EB376 /* iwlwifi-Qu-b0-hr-b0-48.ucode */,
 				F8A68DD92482241E007EB376 /* iwlwifi-QuZ-a0-hr-b0-48.ucode */,
+				17FD7F43255E4EB300611406 /* iwn-100 */,
+				17FD7F3E255E4EB300611406 /* iwn-105 */,
```

Comment:
iwn-100 is not in use, maybe should delete it.

Reply:
Removed. This iwn-100 file come from the firmware package, and apparently it is not being used.